### PR TITLE
Regenerated API docs.

### PR DESCRIPTION
**Pull request by cgranade**

### CODE REVIEW

File: api/canon/Microsoft.Quantum.Canon.yml [hunk 1]
```diff
@@ -1363,24 +1367,24 @@ operations:
 - uid: microsoft.quantum.canon.with
   summary: >-
     Given operations implementing operators `U` and `V`, performs the
-    operation `U†VU` on a target. That is, this operation
+    operation `U�VU` on a target. That is, this operation
```
Comment (tcNickolas) [inline api/canon/Microsoft.Quantum.Canon.yml]: I believe these shouldn't be part of the change; https://github.com/Microsoft/QuantumLibraries/pull/12/files would fix it if merged.

**Comment by alan-geller** [comment]
The QuantumROM function's documentation at https://docs.microsoft.com/en-us/qsharp/api/canon/microsoft.quantum.canon.quantumrom?view=qsharp-preview is rather messed up; I think there's a $ missing, but I'm not sure.

**Comment by alan-geller** [comment]
Also StatePreparationWithComplexCoefficients, at https://docs.microsoft.com/en-us/qsharp/api/canon/microsoft.quantum.canon.statepreparationcomplexcoefficients?view=qsharp-preview.

**Comment by tcNickolas** [comment]
Did the docgen change between the versions? Summary `>-` shows as `|-` now, which makes tracking the relevant changes a little hard.

I'm tracking down the bug in QuantumROM now; found one, one more to go...

**Comment by cgranade** [comment]
@tcNickolas The change from `>-` to `|-` was to correctly format Markdown in summaries, remarks and examples. There was a bug with how YamlDotNet serializes by default that was causing some issues.

**Comment by tcNickolas** [comment]
I've checked that my fixes are in. Do you want to merge https://github.com/Microsoft/QuantumLibraries/pull/12 and regenerate to fix broken Unicode? (for example, https://review.docs.microsoft.com/en-us/qsharp/api/canon/microsoft.quantum.canon.with?view=qsharp-preview&branch=cgranade%2Fupdate-api-docs shows it)
